### PR TITLE
chore: remove deprecated functions to not be used inside the project

### PR DESCRIPTION
**Pull request by aviscode**
Signed-off-by: Avi Rosenberg <avrumi96@gmail.com>

### CODE REVIEW

File: examples/go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/ipfs/go-datastore v0.6.0
 	github.com/ipfs/go-log/v2 v2.5.1
-	github.com/libp2p/go-libp2p v0.24.0
+	github.com/libp2p/go-libp2p v0.24.2
```
Comment (marten-seemann) [inline examples/go.mod:10]: I don't think this dependency update is needed, is it?

Comment (aviscode) [reply examples/go.mod:10]: not really but way not ?

Comment (marten-seemann) [reply examples/go.mod:10]: PRs should be as minimal as possible. If it's not required to update a dependency, it's required to not update a dependency.

Comment (aviscode) [reply examples/go.mod]: @marten-seemann  Done

File: examples/go.mod [hunk 2]
```diff
@@ -59,10 +59,11 @@ require (
 	github.com/libp2p/go-reuseport v0.2.0 // indirect
 	github.com/libp2p/go-yamux/v4 v4.0.0 // indirect
 	github.com/libp2p/zeroconf/v2 v2.2.0 // indirect
-	github.com/lucas-clemente/quic-go v0.31.0 // indirect
+	github.com/lucas-clemente/quic-go v0.31.1 // indirect
```
Comment (marten-seemann) [inline examples/go.mod:62]: There's still a dependency upgrade here.

Comment (aviscode) [reply examples/go.mod]: @marten-seemann  fixed

Comment (marten-seemann) [reply examples/go.mod]: Well, this one. There's another one.

**Comment by marten-seemann** [comment]
To be honest, this PR is costing us more time in repeated reviews than it's actually worth. I'm going to close it.

**Comment by aviscode** [comment]
> To be honest, this PR is costing us more time in repeated reviews than it's actually worth. I'm going to close it.

I don't understand what your problem

**Comment by aviscode** [comment]
I really don't understand your problem
but if u don't know to be nice and pleasant to   pp will not get far in
open-source
this one is getting added when running go mod tidy

**Pull request closed without merging**